src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2931,6 +2931,10 @@ private function registerHtmlSanitizerConfiguration(array $config, ContainerBuil
                 $def->addMethodCall('allowStaticElements', [], true);
             }
 
+            if ($sanitizerConfig['block_body_elements']) {
+                $def->addMethodCall('blockBodyElements', [], true);
```

Comment:
shouldn't this be done *before* the calls to allow safe or static elements, in case both are enabled ?